data/sql/updates/db_world/2022_09_03_01.sql

```diff
@@ -0,0 +1,47 @@
+UPDATE `creature_template` SET `npcflag`='3', `AIName`='SmartAI' WHERE (`entry`='36457');
+UPDATE `creature_template` SET `npcflag`='16777216', `unit_flags`='16384', `AIName`='SmartAI', `flags_extra`='2' WHERE (`entry`='36540');
+UPDATE `creature_template` SET `npcflag`='16777216', `VehicleId`='946', `AIName`='SmartAI', `movementId`='0' WHERE (`entry`='36555');
+UPDATE `creature_template` SET `faction`='2164' WHERE (`entry`='36540');
+INSERT INTO `creature_template` (`entry`, `modelid1`, `name`, `faction`, `speed_walk`, `speed_run`, `scale`, `unit_class`, `unit_flags`, `unit_flags2`, `type`, `AIName`, `movementId`, `flags_extra`, `VerifiedBuild`) VALUES ('365550', '11686', 'Mountain Horse', '2203', '3.6', '2.14286', '0.4', '1', '8', '2048', '1', 'SmartAI', '195', '130', '1');
+INSERT INTO `conditions` VALUES (13, 1, 68916, 0, 0, 31, 0, 3, 36555, 0, 0, 0, 0, '', 'Spell Implicit Target Npc');
```

Comment:
En: whenever an insert is used, it is always accompanied by a delete and you are missing many
ES: siempre que se usa un insert va acompa;ado de un delete y te faltan muchisimos